.github/workflows/build.yml

```diff
@@ -187,17 +187,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        fc: [ifort]
+        fc: [ifx]
         cc: [icx]
     env:
       FC: ${{ matrix.fc }}
       CC: ${{ matrix.cc }}
       WITH_MPI: false
       APT_PACKAGES: >-
-        intel-oneapi-compiler-fortran
-        intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic
-        intel-oneapi-mkl
-        intel-oneapi-mkl-devel
+        intel-basekit
+        intel-hpckit
```

Comment:
That is a real overkill, the basekit contains tons of unnecessary packages.

Also, I've just installed `intel-oneapi-compiler-fortran` on my laptop, and it does install `ifort`. @vanderhe Could you first try just to update the repository keys? Probably that is already enough to fix the issue?

Reply:
Will try, shall we move on to `ifx` then? I would be interested to see how it holds up, but sticking with `ifort` is the safe path, up to you.

Reply:
@aradi Current iteration does not find `ifort` again, am I doing something wrong?